index.bs

```diff
@@ -282,30 +266,164 @@ Loader instances are initially created with the internal slots described in the
   <tr>
     <td>\[[Registry]]</td>
     <td>An object</td>
-    <td>An instance of Registry (<a href="#registry">4</a>).</td>
+    <td>An instance of <a href="#registry">Registry</a>.</td>
   </tr>
 </table>
 
 <h2 id="registry">Registry Objects</h2>
 
-<h3 id="registry-abstract-operations">Abstract Operations for Registry Objects</h3>
+<h3 id="registry-constructor">The Registry Constructor</h3>
+
+The Registry constructor is the %Registry% intrinsic object. When called as a constructor it creates and initializes a new %Registry% object. %Registry% is not intended to be called as a function and will throw an exception when called in that manner.
+
+<h4 id="new-registry" aoid="Registry">Registry()</h4>
+
+When Registry is called with no arguments, the following steps are taken:
+
+<emu-alg>
+1. If NewTarget is *undefined*, then throw a *TypeError* exception.
```

Comment:
We should mention in the conventions section that we're using the same convention for NewTarget as ES6 -- it's described in [9.3: Built-in Function Objects](http://www.ecma-international.org/ecma-262/6.0/index.html#sec-built-in-function-objects). We can add a subsection called Built-in Function Objects and say "We follow the ECMA-262 [convention for built-in function objects](http://www.ecma-international.org/ecma-262/6.0/index.html#sec-built-in-function-objects) in which the value of NewTarget in each function argument is *undefined* for \[\[Call]] and the _newTarget_ parameter for \[\[Construct]]."